react-to-webcomponent.js

```diff
@@ -111,10 +111,10 @@ export default function (ReactComponent, React, ReactDOM, options = {}) {
 			}, this);
 			rendering = true;
 			// Container is either shadow DOM or light DOM depending on `shadow` option.
-			const container = options.shadow ? this.shadowRoot : this;
-
+			const container = options.shadow && options.shadow === 'open' ? this.shadowRoot : this;
```

Comment:
@christopherjbaker I'm not sure if we want to return the shadow root if the mode is closed even if it's null

Reply:
This value is just used internally to render React. Whatever value will render correctly should be good here.